### ISSUE TITLE
Async Tasks DynamoDB in VPC?

### ISSUE THREAD

**Issue by Erstwild** [Miserlou/Zappa#1149]
Is the DynamoDB instance for async task handling created in the VPC specified in "vpc_config"?  Or is that only for the Lambda functions?  If not, what would be the best way to go about configuring that?

**Comment by Miserlou** [comment]
Really great question, @scoates ?

**Comment by scoates** [comment]
Good point. I honestly forgot that DynamoDB could have VPC endpoints. A patch to fix this would be most welcome. (-:

**Comment by Erstwild** [comment]
Thanks for the clarification all.  I have not had a chance to dig into this yet myself, but it sounds like this might simplify adding the feature: https://aws.amazon.com/blogs/aws/new-vpc-endpoints-for-dynamodb/

**Comment by scoates** [comment]
I don't have any lambdas in VPC, right now, but if/when I do, I can take a look at this.
